src/common/Navigation/styled.js

```diff
@@ -1,49 +1,160 @@
 import styled from "styled-components";
+import searchIcon from "./searchIcon.svg";
+import { NavLink } from "react-router-dom";
 
-export const NavigationContainer = styled.div`
-    background-color: ${({ theme }) => theme.colors.navigation.background};
+export const NavigationWrapper = styled.div`
+    background: ${({ theme }) => theme.colors.navigation.background};
     color: ${({ theme }) => theme.colors.navigation.elems};
     min-height: 94px;
+`;
```

Comment:
missing:
max-width: 1920px;
margin: 0 auto;

Reply:
I'm sorry but you're wrong. 

The project is at 1920px but it's not our container. Our website is to be fully responsive. The navigation should be on the width of the page, regardless of the width of the window. With this width, with a higher screen resolution navigation does not cover the entire page.

So your suggestion is wrong, because that's not what we mean.